src/store/messagesStore.js

```diff
@@ -563,6 +563,7 @@ const actions = {
 			const reactionsStore = useReactionsStore()
 			if (message.systemMessage === 'message_deleted') {
```

Comment:
Also need to clean sharedItemsStore for `message.systemMessage === 'history_cleared'`:
- not for yourself, but for other participants
- same as in messagesStore, it should not remove newer messages (`messageToRemoveId < systemMessage .id`)